pkg/custom_detectors/custom_detectors.go

```diff
@@ -263,3 +297,20 @@ func (c *CustomRegexWebhook) Description() string {
 	}
 	return c.GetDescription()
 }
+
+// helper function to handle JSON response
+func handleJSONResponse(body []byte) (string, error) {
+	var respBody interface{}
+	err := json.Unmarshal(body, &respBody)
+	if err != nil {
+		return "", fmt.Errorf("failed to unmarshal JSON: %v", err)
+	}
+
+	// convert JSON map to a formatted string
+	jsonString, err := json.MarshalIndent(respBody, "", "  ")
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal JSON: %v", err)
+	}
+
+	return strings.TrimSpace(string(jsonString)), nil
+}
```

Comment:
Why does this function unmarshal then marshal?

Reply:
Ah, I think I might have over-engineered this, even though @ahrav warned me not to! 😅 I was trying to get the response into a nicely formatted JSON string, but I realize now that it's not really necessary. We can just stick to simple string conversion instead.